atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/ranges/DbKvsGetRanges.java

```diff
@@ -191,16 +191,16 @@ public DbKvsGetRanges(DbKvs kvs,
             String query = String.format(
                     SIMPLE_ROW_SELECT_TEMPLATE,
                     tableName,
```

Comment:
This is how it is done internally (not a reason to do it externally, but is why I kept it this way).  In addition is also does make sense from a certain perspective -- namely the user says "I want to add table 'hello_world'" then the sql tracing comment will say you were creating table 'hello_world' while secretly doing the prefixing to provide safety.  At the same time I see the argument for having transparency in what is actually stored.